modules/swagger-codegen-cli/src/main/java/io/swagger/codegen/cmd/Generate.java

```diff
@@ -74,6 +74,10 @@
             description = CodegenConstants.API_PACKAGE_DESC)
     private String apiPackage;
 
+    @Option(name = {"--api-name-suffix"}, title = "api name suffix",
```

Comment:
I don't think this should be here at all. If I'm not mistaken, these are the general swagger-codegen wide parameters that all generator should support. I think you only need to add you parameters to your specific codegen. See this for an example: 

https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaPlayFrameworkCodegen.java#L22

Reply:
In fact, I check all your code, it seems you are indeed trying to create an "all generators" parameter... I'm not sure it is really safe to add that kind of parameters. I will add @wing328  and @cbornet to the conversation to have their advices.